independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -635,9 +1078,778 @@
+                  "title": "archived"
+                },
+                "properties": {
+                  "type": "object",
+                  "properties": {},
+                  "description": "Property values of this page.",
+                  "title": "Page properties"
+                },
+                "url": {
+                  "type": "string",
+                  "description": "The URL of the Notion page.",
+                  "title": "URL"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create a page",
+        "description": "Creates a new page in the specified database. existing page.",
+        "operationId": "CreateaPage",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Notion-Version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "2022-06-28",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "parent": {
+                  "type": "object",
+                  "properties": {
+                    "database_id": {
+                      "type": "string",
+                      "description": "Unique identifier for the database.",
+                      "title": "Database ID",
+                      "x-ms-visibility": "important"
+                    }
+                  },
+                  "description": "parent",
+                  "required": [
+                    "database_id"
+                  ]
+                },
+                "icon": {
+                  "type": "object",
+                  "properties": {
+                    "emoji": {
+                      "type": "string",
+                      "description": "emoji",
+                      "title": "Emoji"
+                    }
+                  },
+                  "description": "icon"
+                },
+                "cover": {
+                  "type": "object",
+                  "properties": {
+                    "external": {
+                      "type": "object",
+                      "properties": {
+                        "url": {
+                          "type": "string",
+                          "description": "Page cover image URL.",
+                          "title": "Cover URL",
+                          "x-ms-visibility": "important"
+                        }
+                      },
+                      "description": "external"
+                    }
+                  },
+                  "description": "Page cover image.	"
```

Comment:
@sekharmalla Please remove extra spaces from the any value in the swagger.

Reply:
I removed Extra space from the description. I will double-check again. but @sriyen-msft  can you advise me on how do I fix the conflict error? 

I have updated the same changes on my original branch also.